### PR TITLE
Task automation and test improvement

### PR DESCRIPTION
**Pull request by unlikelyzero**
Closes: #4423 

### Describe your changes:
This ticket attempts to address some of the issues we've been having with missed or slipped comments or testing notes. As the team has grown, our change velocity has increased  -- and so have the associated number of PRs/Issues.

In the first implementation, we will not be blocking PRs -- just informing of failure.

Note: there is currently a bug with prcop which prevents it from picking up changes to the main config file on branches vs master. That is being tracked here https://github.com/makaroni4/prcop/issues/3

This will work after merge to master

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [x] Is this change backwards compatible? For example, developers won't need to change how they are calling the API or how they've extended core plugins such as Tables or Plots.

### Author Checklist

* [x] Changes address original issue?
* [x] Unit tests included and/or updated with changes?
* [x] Command line build passes?
* [x] Has this been smoke tested?
* [ ] Testing instructions included in associated issue?

### Reviewer Checklist
* [ ] Changes appear to address issue?
* [ ] Changes appear not to be breaking changes?
* [ ] Appropriate unit tests included?
* [ ] Code style and in-line documentation are appropriate?
* [ ] Commit messages meet standards?
* [ ] Has associated issue been labelled unverified? (only applicable if this PR closes the issue)
* [ ] Has associated issue been labelled bug? (only applicable if this PR is for a bug fix)
* [ ] List of Acceptance Tests Performed. These will be turned into e2e tests.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -11,3 +17,14 @@
...
+### Reviewer Checklist
+
+* [ ] Changes appear to address issue?
+* [ ] Changes appear not to be breaking changes?
+* [ ] Appropriate unit tests included?
+* [ ] Code style and in-line documentation are appropriate?
+* [ ] Commit messages meet standards?
+* [ ] Has associated issue been labelled unverified? (only applicable if this PR closes the issue)
+* [ ] Has associated issue been labelled bug? (only applicable if this PR is for a bug fix)
+* [ ] List of Acceptance Tests Performed. These will be turned into e2e tests.
```
Comment (akhenry) [inline .github/PULL_REQUEST_TEMPLATE.md:30]: Could we remove this for now just to get the automation stuff in? The specifics of per-PR testing are currently under review.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+<!--- Note: Please open the PR in draft form until you are ready for active review. -->
+Closes: <!--- Insert Issue Number(s) this PR addresses. Typing # will open a dropdown of recent issues. -->
```
Comment (akhenry) [inline .github/PULL_REQUEST_TEMPLATE.md:2]: I'm not sure if the GitHub parser likes the colon here. I recently [merged a PR with `"resolves: #...."`](https://github.com/nasa/openmct/pull/4475) and it didn't automatically close the linked issue as expected, even though "resolves" is one of [GitHub's supported keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) . I had to close it manually. I've never had this issue without the colon (unless something else was wrong, like a typo in the keyword).

**Review by akhenry — CHANGES_REQUESTED** [review]
This looks really awesome, and adds a **lot** of value. Two small suggestions inline. I also want to make sure @jvigliotta takes a look at this.

**Comment by jvigliotta** [comment]
@unlikelyzero Maybe not a check-box, but can we mention something to the effect of "Any outside contributors must have a signed CLA on file."?

**Comment by unlikelyzero** [comment]
@jvigliotta spun that out to automation here https://github.com/nasa/openmct/issues/4739

**Review by shefalijoshi — APPROVED** [review]
LGTM
### Reviewer Checklist

* [ ] Changes appear to address issue?
* [ ] Changes appear not to be breaking changes?
* [ ] Appropriate unit tests included?
* [ ] Code style and in-line documentation are appropriate?
* [ ] Commit messages meet standards?
* [ ] Has associated issue been labelled `unverified`? (only applicable if this PR closes the issue)
* [ ] Has associated issue been labelled `bug`? (only applicable if this PR is for a bug fix)
* [ ] List of Acceptance Tests Performed.

**Review by akhenry — APPROVED** [review]

**Pull request merged**